crates/fda/src/cli.rs

```diff
@@ -43,7 +43,10 @@ fn pipeline_names(current: &std::ffi::OsStr) -> Vec<CompletionCandidate> {
 pub struct Cli {
     #[command(subcommand)]
     pub command: Commands,
-    /// The format in which the output should be displayed.
+    /// The format in which the outputs from feldera should be displayed.
+    ///
+    /// Note that this flag may have no effect on some commands in case
+    /// the requested output format is not supported for it.
```

Comment:
you can delete "for it".

Reply:
true but don't want to go through CI again just for deleting two words